masterlist.yaml

```diff
@@ -8310,6 +8302,16 @@ plugins:
     msg:
       - <<: *patchOutSub
         condition: 'not checksum("SeraphimStandAlone.esp",556E0B68)'
+  - name: 'SeraphineHuntedArmor.esp'
+    msg:
+      - type: say
+        content:
+          - lang: en
+            text: 'Be sure you have the correct body mod for the esp. [DIMONIZED UNP](http://www.nexusmods.com/skyrim/mods/6709) or [CBBE](http://www.nexusmods.com/skyrim/mods/2666).'
+          - lang: ru
+            text: 'Убедитесь, что используете корректный реплейсер тел для этого esp. [DIMONIZED UNP](http://www.nexusmods.com/skyrim/mods/6709) или [CBBE](http://www.nexusmods.com/skyrim/mods/2666).'
+          - lang: ja
+            text: '必ず[DIMONIZED UNP](http://www.nexusmods.com/skyrim/mods/6709)か[CBBE](http://www.nexusmods.com/skyrim/mods/2666)のうち、インストールされている体型Modと同じバージョンのEspを使用してください。.'
```

Comment:
I believe this was to ensure they had a body mod installed.
As I'm not aware of any way to detect if these are installed.